frontend/src/layouts/AppLayout/AppLayout.tsx

```diff
@@ -143,6 +146,23 @@ export const AppLayout = ({ children }: LayoutProps) => {
 
   const { t } = useTranslation();
 
+  const registerUserAction = useRegisterUserAction();
+
+  useEffect(async () => {
```

Comment:
Put a dependency or even empty one. This gets fired on each render

Reply:
Also why not move this also to react-query.

Reply:
Agree this should be moved to `react-query`; I'll make the adjustment.

Reply:
Updated.